group_vars/all.yml

```diff
@@ -140,3 +140,36 @@ firsttime_connect_user: "customeradmin"
 control_node_key_file: "~/.ssh/id_rsa_oracle_toolkit"
 proxy_setup: "{{ lookup('env','ORA_PROXY_SETUP')|lower|default('false',true) }}"
 u01_lun: "{{ lookup('env','ORA_U01_LUN')|lower|default('',true) }}"
+
+#  Validations
+minimum_ansible_version: 2.9
+os_family_supported: "RedHat"
+os_min_supported_version: "{% if free_edition %}8{% else %}7.3{% endif %}"
+os_supported_architecture: "x86_64"
+os_minimum_memory_mb: 2560
+
+control_node_checks:
```

Comment:
Given that these are logic-heavy, can we put these checks into a playbook and include as necessary?

This file is a good place for constants though.

Reply:
btw I like how easy these are to read, and concisely written.